.dockerignore

```diff
@@ -1,3 +0,0 @@
-*
-!build
```

Comment:
I missed that ignoring the `build` directory is required because if `make build` runs before `make image`,
the built artifacts are copied into a container, so binaries that should be built in the container are never built.

Reply:
I missed too. Thanks.